config.yaml

```diff
@@ -1,5 +1,7 @@
 global:
   vars:
+    Reserve: '0x9380fA34Fd9e4Fd14c06305fd7B6199089eD4eb9'
```

Comment:
not related to this PR but do you think we should just keep a single yaml config file? feels weird to be basically duplicating it all the time 😅

Reply:
fair point, tbh i never questioned this myself, can't think of a good reason to have separate files.

@bayological anything speaking against removing `config.local.yml`?

Reply:
I agree with @nvtaveras here. I see no reason to keep it around.

Reply:
aight, it's gone!